src/how-to/create-dates.md

```diff
@@ -0,0 +1,87 @@
+# Create dates and times
+
+`time` provides a [`const`](https://doc.rust-lang.org/std/keyword.const.html)-ready API. All 
+functions here are `const`: values are calculated at compile time, not at runtime.
+
+For convenience, macros are provided with
+[feature `macros`](https://docs.rs/time/latest/time/#feature-flags). They let us restrict
```

Comment:
Let's put the links on their own lines. so `` [feature `macros`][macros] `` and `[macros]: https://docs.rs/time/latest/time/#feature-flags`. This deduplicates anything that is present more than once, and also makes it a bit easier to read the raw markdown.